lib/test/structuredfield/isId3Footer.test.ts

```diff
@@ -0,0 +1,27 @@
+import { equal } from 'node:assert';
+import { describe, it } from 'node:test';
+import { isId3Footer } from '../../src/id3/util/isId3Footer.js';
+
+describe('isId3Footer', () => {
+	const mockID3Footer = Uint8Array.from([
+		0x33, 0x44, 0x49, 4, 0, 0, 0, 0, 0, 63, 80, 82, 73, 86, 0, 0, 0, 53, 0, 0,
+		99, 111, 109, 46, 97, 112, 112, 108, 101, 46, 115, 116, 114, 101, 97, 109,
+		105, 110, 103, 46, 116, 114, 97, 110, 115, 112, 111, 114, 116, 83, 116, 114,
+		101, 97, 109, 84, 105, 109, 101, 115, 116, 97, 109, 112, 0, 0, 0, 0, 0, 0,
+		13, 198, 135,
+	]);
+	const mockID3FooterMissingLeadingByte = mockID3Footer.slice(
+		8,
+		mockID3Footer.length
+	);
+	const mockID3FooterMissingTrailingByte = mockID3Footer.slice(
+		0,
+		mockID3Footer.length - 8
```

Comment:
what exactly is 8? I dont' understand what this magic number means.

Reply:
my understanding is that the footer is set on the last bytes of the Id3 tags, so the length is for it to know where to look for the footer,  the number was already set on the original test, so i left it as it was.

Reply:
My guess on this one is because it's actually removing the last byte, by the name of the const (`mockID3FooterMissingTrailingByte`).

If you take a look the at the above (`mockID3FooterMissingLeadingByte`), it starts at 8, because it's missing the leading byte.